src/daemon/org.projectatomic.rpmostree1.conf

```diff
@@ -37,5 +37,9 @@
     <allow send_destination="org.projectatomic.rpmostree1"
            send_interface="org.projectatomic.rpmostree1.Sysroot"
            send_member="GetOS"/>
+
+    <allow send_destination="org.projectatomic.rpmostree1"
+           send_interface="org.projectatomic.rpmostree1.Sysroot"
+           send_member="RegisterClient"/>
```

Comment:
Maybe we also at least have this method require being logged in?  In a world of containers this matters a lot less, but it's always been weird to have e.g. httpd be able to call methods like this.